subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/ResolutionErrorRenderer.java

```diff
@@ -63,38 +64,32 @@ private void handleResolutionError(Throwable cause) {
     }
 
     private void handleOutOfDateLocks(final LockOutOfDateException cause) {
-        registerError(new Action<StyledTextOutput>() {
-            @Override
-            public void execute(StyledTextOutput output) {
-                List<String> errors = cause.getErrors();
-                output.text("The dependency locks are out-of-date:");
-                output.println();
-                for (String error : errors) {
-                    output.text("   - " + error);
-                    output.println();
-                }
+        registerError(output -> {
+            List<String> errors = cause.getErrors();
+            output.text("The dependency locks are out-of-date:");
+            output.println();
+            for (String error : errors) {
+                output.text("   - " + error);
                 output.println();
             }
+            output.println();
```

Comment:
Was this introduced intentionally?

Reply:
Checking

Reply:
The diff rendering is broken. That `output.println()` was present inside the anonymous inner class already. So it needs to remain.
Same for the next diff section btw.